### PR TITLE
Zeek known certs grok fix

### PR DESCRIPTION
**Pull request by peterydzynski**
## Proposed commit message
The `grok` processors that extract the Common Name from the subject and issuer fields in TLS certificates fails when the Common Name is not present. While not valid, this can still happen in a few cases like self signed certificates. To solve this I added a condition to the processor to check that `CN=` is present in the source field before executing the processor. 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- 

## Screenshots
known-certs

### CODE REVIEW

File: packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -104,6 +104,7 @@ processors:
         - 'CN=%{CN:tls.server.x509.subject.common_name}'
       pattern_definitions:
         CN: '[^,]+'
+      if: ctx?.tls?.server?.x509?.subject?.distinguished_name != null && ctx.tls.server.x509.subject.distinguished_name.contains('CN=')
```
Comment (andrewkroh) [inline packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml:107]:
```suggestion
      if: ctx.tls?.server?.x509?.subject?.distinguished_name != null && ctx.tls.server.x509.subject.distinguished_name.contains('CN=')
```

`ctx` is never null.

File: packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -115,6 +116,7 @@ processors:
         - 'CN=%{CN:tls.server.x509.issuer.common_name}'
       pattern_definitions:
         CN: '[^,]+'
+      if: ctx?.tls?.server?.x509?.issuer?.distinguished_name != null && ctx.tls.server.x509.issuer.distinguished_name.contains('CN=')
```
Comment (andrewkroh) [inline packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml:119]:
```suggestion
      if: ctx.tls?.server?.x509?.issuer?.distinguished_name != null && ctx.tls.server.x509.issuer.distinguished_name.contains('CN=')
```

**Comment by peterydzynski** [comment]
Thanks for the edits @andrewkroh!

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**